packages/teams-js/src/internal/utils.ts

```diff
@@ -6,6 +6,7 @@ import { GlobalVars } from '../internal/globalVars';
 import { minAdaptiveCardVersion } from '../public/constants';
 import { AdaptiveCardVersion, SdkError } from '../public/interfaces';
 import { pages } from '../public/pages';
+import { Communication, sendAndUnwrap } from './communication';
```

Comment:
I am concerned about creating a circular dependency between utils.ts and communication.ts. They each import each other.

In my experience, this is one of those things which sometimes seems to work but then randomly breaks later when someone makes an unrelated change because some internal compiler thing gets tripped.

Is it possible to extract some of this code into a different place to avoid creating the circular dependency?